### ISSUE TITLE
Hello, excellent work! If my primary focus is on depth accuracy and I'm working on dynamic scenes, would it be feasible to omit the Gaussian splatting process in the later stages?

### ISSUE THREAD

**Issue by booker-max** [cvg/depthsplat#41]
Hello, this is a fantastic idea!
 I have a question: If I am only concerned with the performance of depth estimation and do not need to perform novel view synthesis, I want to conduct large-scale training on my own dataset. 
My dataset primarily consists of dynamic scenes in autonomous driving scenarios, which suggests that Gaussian splatting might not be effective for further enhancing the depth estimation model.
 Instead, I plan to use only your proposed mono-MVS fusion strategy for this task. 
I suspect this approach would handle dynamic objects well due to the injection of monocular depth features.
 Do you think this is feasible? Or do you have other suggestions?

**Comment by haofeixu** [comment]
Yes, this is feasible and our depth model could also work for dynamic scenes thanks to the monocular features.

**Comment by haofeixu** [comment]
Hi, additional fine-tuning would be necessary since the pre-trained models never see fisheye images.
